transformation/src/jade-schema/tables/human_donor.table.json

```diff
@@ -48,8 +47,7 @@
     },
     {
       "name": "life_stage",
-      "datatype": "string",
-      "type": "required"
+      "datatype": "string"
     },
     {
       "name": "parent_ids",
```

Comment:
I realized that "age" and "life_stage" are not always defined in the encode data, so I am making them optional here. I ran this change by Kathy already.